packages/cli/src/commands/network/whitelist.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseCommand } from '../../base'
+
+export default class Whitelist extends BaseCommand {
```

Comment:
oh this is nicer than adding to the parameters anyway

Reply:
🙇